app/lib/common/widgets/square_avatar.dart

```diff
@@ -0,0 +1,32 @@
+import 'package:flutter/material.dart';
+
+class SquareAvatar extends StatelessWidget {
```

Comment:
This widget can also be discarded since we are using ```acter_avatar``` standalone package for handling acter avatars in general.

Reply:
Noted